### ISSUE TITLE
Clarify guidelines for video use

### ISSUE THREAD

**Issue by timfrick** [w3c/sustainableweb-wsg#4]
This suggestion was given to me by an interaction designer for UK gov on LinkedIn regarding video:

> The immediate one I think of relates to video which is helpful for people who struggle with written text and as relay necessary for Sign language interpretation on calls. You are right that your guidance doesn't contradict, however I notice that it doesn't mention it either? 

2.16 does specifically mention accessibility benefits:

> Delivering media assets in ways that convey information in an easy-to-read manner both visually and contextually (even if people are unable to for example see), will allow a wider audience to gain from your content.

However, to her point, I'm a little hung up on the phrasing 'easy to read'. Shouldn't that also be easy to perceive, understand, etc. per WCAG? Are there opportunities to be more specific and clear here, especially regarding the use case she mentions?

**Comment by TzviyaSiegman** [comment]
we might consider cross-referencing wcag throughout

**Comment by AlexDawsonUK** [comment]
@timfrick I agree easy-to-perceive is a better choice of word use. I've made the adjustment.

I have also improved the language in [2.16](https://w3c.github.io/sustainableweb-wsg/#success-criterion-need-for-media-machine-testable) for SC Need For Media
> The need for video or sound usage (including only when they add visitor value) has been determined

In cases like sign language, this is adding visitor value (accessibility) so it is self-evident that its meeting the criteria.

@andreadavanzo We mentioned in the meeting about having better ability to filter / group the guidelines & Success Criteria by applicability (that would offer the global style appeal you are looking for, I've made a note of this for future reference).
